### ISSUE TITLE
Don't work online account in elementary switchboard

### ISSUE THREAD

**Issue by ivan-tretyak** [decathorpe/elementary-nightly-rpms#45]
I can't use the online accounts for elementary email or scrobbler my music to LastFM.
`[twilight@equestria]>07:07:14> io.elementary.switchboard 

(io.elementary.switchboard:5044): GLib-CRITICAL **: 19:18:26.997: g_variant_get_type: assertion 'value != NULL' failed

(io.elementary.switchboard:5044): GLib-CRITICAL **: 19:18:26.997: g_variant_type_is_subtype_of: assertion 'g_variant_type_check (type)' failed

(io.elementary.switchboard:5044): GLib-CRITICAL **: 19:18:26.997: g_variant_dup_strv: assertion 'g_variant_is_of_type (value, G_VARIANT_TYPE_STRING_ARRAY)' failed

** (io.elementary.switchboard:5044): CRITICAL **: 19:18:27.906: LastFMProvider.vala:67: GDBus.Error:com.google.code.AccountsSSO.gSingleSignOn.Error.MethodNotKnown: authentication method 'lastfm' doesn't exist

** (io.elementary.switchboard:5044): CRITICAL **: 19:18:50.626: PasswordProvider.vala:74: GDBus.Error:com.google.code.AccountsSSO.gSingleSignOn.Error.PermissionDenied: identity can not be accessed

** (io.elementary.switchboard:5044): CRITICAL **: 19:18:51.313: OAuthProvider.vala:81: GDBus.Error:com.google.code.AccountsSSO.gSingleSignOn.Error.PermissionDenied: identity can not be accessed

** (io.elementary.switchboard:5044): CRITICAL **: 19:18:52.051: OAuthProvider.vala:81: GDBus.Error:com.google.code.AccountsSSO.gSingleSignOn.Error.PermissionDenied: identity can not be accessed

** (io.elementary.switchboard:5044): CRITICAL **: 19:18:52.799: OAuthProvider.vala:81: GDBus.Error:com.google.code.AccountsSSO.gSingleSignOn.Error.PermissionDenied: identity can not be accessed

** (io.elementary.switchboard:5044): CRITICAL **: 19:18:53.588: EmailProvider.vala:65: GDBus.Error:com.google.code.AccountsSSO.gSingleSignOn.Error.PermissionDenied: identity can not be accessed

** (io.elementary.switchboard:5044): CRITICAL **: 19:18:54.256: OAuthProvider.vala:81: GDBus.Error:com.google.code.AccountsSSO.gSingleSignOn.Error.PermissionDenied: identity can not be accessed
`

**Comment by decathorpe** [comment]
Side note, the Last.FM scrobbler plugin for Music isn't even enabled in fedora (yet), and I'll only do that for fedora 30 and later.

Because the Last.FM plugin for gsignond is only available on fedora 30 and later.

**Comment by ivan-tretyak** [comment]
Оk. But I can't use Google too.

**Comment by decathorpe** [comment]
Yes, I can confirm that online accounts doesn't work at all on fedora 29 ...

That's strange, because it used to work when I introduced those packages to fedora.

However, all accounts work on fedora 30/rawhide, so I guess it is probably caused by the recent DBus interface changes in gsignond and its plugins.
